no-browser-site-scrapers/ZocDoc/index.js

```diff
@@ -32,7 +36,6 @@ async function ScrapeWebsiteData(fetchService) {
             ...sites[value[0]],
             ...value[1],
             hasAvailability: Object.keys(value[1].availability).length > 0,
-            timestamp: moment().format(),
```

Comment:
Global comment... Could these all be the same? Either  `moment().format()` or `new Date()`